test/integration-tests/UniversalRouter.test.ts

```diff
@@ -242,7 +242,7 @@ describe('UniversalRouter', () => {
 
         const wethAmount = ethers.utils.parseEther('1')
 
-        planner.addCommand(CommandType.WRAP_ETH, [alice.address, wethAmount])
+        planner.addCommand(CommandType.WRAP_ETH, [router.address, wethAmount])
```

Comment:
I originally had
planner.addCommand(CommandType.WRAP_ETH, [alice.address, wethAmount])
but I believe it is ideal to send the wrapped ETH to the router before using it to swap for DAI. Can anyone confirm?